### DISCUSSION TITLE
Error when pod install for ios

### DISCUSSION THREAD

**Discussion by NadeemHungryHop** [flutter-stripe/flutter_stripe#1899] [Q&A]
[!] Error installing Stripe
[!] /usr/bin/git clone https://github.com/stripe/stripe-ios.git /var/folders/_4/3wmsrr517t3b11wgzg18n8c00000gn/T/d20240822-11320-1tymn96 --template= --single-branch --depth 1 --branch 23.28.1

**Comment by basnetjiten** [comment]
Got the same issue: git clone https://github.com/stripe/stripe-ios.git /var/folders/97/6kbwkqvd1rd3kyytw42scbd40000gp/T/d20240919-17743-pc6dfl --template= --single-branch --depth 1 --branch 23.28.3
Cloning into '/var/folders/97/6kbwkqvd1rd3kyytw42scbd40000gp/T/d20240919-17743-pc6dfl'...
remote: Enumerating objects: 7506, done.
remote: Counting objects: 100% (7506/7506), done.
remote: Compressing objects: 100% (4856/4856), done.
error: RPC failed; curl 92 HTTP/2 stream 5 was not closed cleanly: CANCEL (err 8)
error: 4648 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output

**Reply by basnetjiten** [reply]
I was able to resolve it with these commands:
brew upgrade git
gem install cocoapods-clean
pod deintegrate
pod install

**Reply by BunnyBuddy** [reply]
didn't solve the problem for me
